web/Views/Admin/PropertyList.cshtml

```diff
@@ -7,42 +7,56 @@
 <h2>All Properties List</h2>
 
 <div class="mb-3">
-    <div class="row">
-        <div class="col-4">
-            @using (Html.BeginForm("SortProperties", "Property", FormMethod.Get))
-            {
-                <select class="form-select" id="sortBy" name="sortBy">
-                    <option value="Availability">Sort by availability</option>
-                    <option value="Rent">Sort by rent</option>
-                </select>
-                <button type="submit" class="btn custom-btn-primary"> Display results </button>
-            }
-        </div>
-        <div class="col-8">
-            @using (Html.BeginForm("SortPropertiesByLocation", "Property", FormMethod.Get))
-            {
-                <input class="form-control" id="postcode" name="postcode" required placeholder="Enter postcode to sort by location" pattern="([Gg][Ii][Rr] 0[Aa]{2})|((([A-Za-z][0-9]{1,2})|(([A-Za-z][A-Ha-hJ-Yj-y][0-9]{1,2})|(([A-Za-z][0-9][A-Za-z])|([A-Za-z][A-Ha-hJ-Yj-y][0-9][A-Za-z]?))))\s?[0-9][A-Za-z]{2})"/>
-                <button type="submit" class="btn custom-btn-primary"> Sort </button>
-            }
+    @using (Html.BeginForm("PropertyList", "Property", FormMethod.Get))
+    {
+        <div class="row">
+            <select class="form-select" name="sortBy"

```

Comment:
Could we use the dropdown list here instead of the select :)

Reply:
Max is sorting this on his branch - we agreed that there's no point duplicating the work.